openstack-mixin/panels.libsonnet

```diff
@@ -277,10 +260,61 @@ local utils = commonlib.utils;
               openstack_placement_resource_total: 'Total',
               openstack_placement_resource_usage: 'In use',
               resourcetype: 'Resource',
+              'cluster 1': 'cluster',
             },
             includeByName: {},
           },
         },
+        {
+          id: 'filterFieldsByName',
+          options: {
+            include: {
+              pattern: 'Disk.*|Memory.*|VCPU.*|cluster',
+            },
+          },
+        },
+      ]),
+
+    vCPUusedStat:
+      gauge.new(
+        'vCPU used'
+      )
+      + gauge.queryOptions.withTargetsMixin(t.vCPUused)
+      + gauge.standardOptions.withUnit('percent')
+      + gauge.standardOptions.withMin('0')
+      + gauge.standardOptions.withMax('150')
+      + gauge.standardOptions.thresholds.withSteps([
+        gauge.standardOptions.threshold.step.withValue(0) +
+        gauge.standardOptions.threshold.step.withColor('green'),
+        gauge.standardOptions.threshold.step.withValue(99) +
+        gauge.standardOptions.threshold.step.withColor('red'),
+      ]),
+
+    RAMusedStat:
+      gauge.new(
+        'Memory used'
+      )
+      + gauge.queryOptions.withTargetsMixin(t.RAMused)
+      + gauge.standardOptions.withUnit('percent')
+      + gauge.standardOptions.withMin('0')
+      + gauge.standardOptions.withMax('150')
+      + gauge.standardOptions.thresholds.withSteps([
+        gauge.standardOptions.threshold.step.withValue(0) +
+        gauge.standardOptions.threshold.step.withColor('green'),
+        gauge.standardOptions.threshold.step.withValue(99) +
+        gauge.standardOptions.threshold.step.withColor('red'),
+      ]),
+
+    freeIPsStat:
```

Comment:
![image](https://github.com/grafana/jsonnet-libs/assets/69878936/2e581d57-87bf-4207-b272-ccdc7208e492)
This dashboard could use a little styling to be more readable